### PR TITLE
cp-342: Modify response for meditation post

### PR DESCRIPTION
**Pull request by OlesiaTr**
- [x]  Added 'id' to the response for 201 status code
- [x]  Added to swagger documentation sample responses for incorrect format and for file size over 10Mb.

adding_id

### CODE REVIEW

**Review by liza-veis — DISMISSED** [review]

File: backend/src/packages/meditation/meditation.service.ts [hunk 1]
```diff
@@ -43,10 +43,11 @@ class MeditationService implements Service {
     name: string;
     file: FileUploadRequestDto;
   }): Promise<MeditationEntryCreateResponseDto> {
-    const { url, contentType } = await this.fileService.create(file);
+    const { id, url, contentType } = await this.fileService.create(file);
     const item = await this.meditationRepository.create(
       MeditationEntity.initializeNew({
-        name: name,
+        id,
```
Comment (fshabanov) [inline backend/src/packages/meditation/meditation.service.ts]: Why are you using `id` of the file to create a meditation?

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Review by OlenaYatseniuk — COMMENTED** [review]
Can you also add, please, security to post route as it is in get route?

File: backend/src/packages/meditation/meditation.service.ts (hunk 1 above)
Comment (OlesiaTr) [reply backend/src/packages/meditation/meditation.service.ts]: Because I thought they should be added to the meditation entries

Comment (liza-veis) [reply backend/src/packages/meditation/meditation.service.ts]: Farid is right, id's are created automatically, you task is to return it in response not to set it

**Review by liza-veis — CHANGES_REQUESTED** [review]

File: backend/src/packages/meditation/meditation.service.ts (hunk 1 above)
Comment (OlesiaTr) [reply backend/src/packages/meditation/meditation.service.ts]: Yes, sorry, you are correct, I got it misunderstood

**Review by liza-veis — APPROVED** [review]

**Review by fshabanov — APPROVED** [review]

**Review by HIMDanny — APPROVED** [review]

**Pull request merged**